src/Engine/AttackList.cpp

```diff
@@ -0,0 +1,17 @@
+#include "Engine/AttackList.h"
+#include "Engine/Engine.h"
+
+std::vector<AttackDescription> attackList;
+
+// Original function was replaced
+//----- (0040261D) --------------------------------------------------------
+// void stru298::Add(int16_t uID, int16_t a3, int16_t x, int16_t y, int16_t z, ABILITY_INDEX a7, char a8);
```

Comment:
If one of the functions below does conceptually the same thing, you can just add @offset to one of them.